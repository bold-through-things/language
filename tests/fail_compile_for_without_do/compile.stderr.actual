[
  {
    "recoverable": false,
    "line": 2,
    "char": 0,
    "content": "for i in",
    "error": "failed to assert: expected 'do' after 'for' but found 'then'",
    "error_type": "EXPECTED_DO_AFTER"
  },
  {
    "recoverable": false,
    "line": 9,
    "char": 0,
    "content": "67lang:access_local i",
    "error": "failed to assert: i must access a defined local",
    "error_type": "NO_SUCH_LOCAL"
  },
  {
    "recoverable": false,
    "line": 2,
    "char": 0,
    "content": "for i in",
    "error": "failed to assert: must have a `do` block",
    "error_type": "ASSERTION_FAILED"
  }
]
