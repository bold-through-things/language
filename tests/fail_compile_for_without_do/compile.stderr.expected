[
  {
    "recoverable": false,
    "line": 2,
    "char": 0,
    "content": "for i in",
    "error": "failed to assert: expected 'do' after 'for' but found 'then'",
    "error_type": "EXPECTED_DO_AFTER"
  },
  {
    "recoverable": false,
    "line": 10,
    "char": 0,
    "content": "67lang:call i",
    "error": "failed to assert: could not find a matching overload for i with arguments []",
    "error_type": "NO_MATCHING_OVERLOAD",
    "visible_overloads": [
      {
        "field": "i",
        "demands": [
          "HTMLElementTagNameMap"
        ],
        "returns": {
          "name": "HTMLElement",
          "type_params": [],
          "fields": []
        },
        "convention": "FieldCall"
      },
      {
        "field": "i",
        "demands": [
          "HTMLElementTagNameMap",
          {
            "name": "HTMLElement",
            "type_params": [],
            "fields": []
          }
        ],
        "returns": "None",
        "convention": "FieldCall"
      }
    ]
  },
  {
    "recoverable": false,
    "line": 2,
    "char": 0,
    "content": "for i in",
    "error": "failed to assert: must have a `do` block",
    "error_type": "ASSERTION_FAILED"
  },
  {
    "recoverable": false,
    "line": 10,
    "char": 0,
    "content": "67lang:call i",
    "error": "failed to assert: could not find a matching overload for i with arguments None",
    "error_type": "NO_MATCHING_OVERLOAD",
    "visible_overloads": [
      {
        "field": "i",
        "demands": [
          "HTMLElementTagNameMap"
        ],
        "returns": {
          "name": "HTMLElement",
          "type_params": [],
          "fields": []
        },
        "convention": "FieldCall"
      },
      {
        "field": "i",
        "demands": [
          "HTMLElementTagNameMap",
          {
            "name": "HTMLElement",
            "type_params": [],
            "fields": []
          }
        ],
        "returns": "None",
        "convention": "FieldCall"
      }
    ]
  }
]
